dms_storage/models/dms_file.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 CreuBlanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class DmsFile(models.Model):
+    _inherit = "dms.file"
+
+    storage_path = fields.Char(invisible=True)
```

Comment:
Invisible not affect in model only in view. Maybe is much better to put readonly field and invisible it in view